cmd/support-perf.go

```diff
@@ -65,6 +65,10 @@ var supportPerfFlags = append([]cli.Flag{
 		Usage:  "do not clear bucket after running object perf test",
 		Hidden: true, // Hidden for now.
 	},
+	cli.BoolFlag{
+		Name:  "enable-sha256-payload",
```

Comment:
```suggestion
		Name:  "sha256",
```

Let's keep flags short and easy to use.